translations/fr.po

```diff
@@ -11,91 +11,112 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Poedit 3.1.1\n"
+"X-Generator: Poedit 3.2.2\n"
 "X-Poedit-Basepath: ..\n"
 "X-Poedit-SearchPath-0: .\n"
 
-#: config.py:82 config.py:324
+#: config.py:87 config.py:353
 msgid "Preferred format order"
 msgstr "Ordre de préférence des formats"
 
-#: config.py:86 custom_lemmas.py:53
+#: config.py:91 custom_lemmas.py:56
 msgid "Customize Kindle Word Wise"
 msgstr "Personaliser Kindle Word Wise"
 
-#: config.py:90
+#: config.py:95
 msgid "Customize EPUB Wiktionary"
 msgstr "Personaliser EPUB Wiktionary"
 
-#: config.py:95
+#: config.py:100
 msgid "Fetch X-Ray people descriptions from Wikipedia/Fandom"
 msgstr "Chercher les descriptions de personnes X-Ray depuis Wikipedia/Fandom"
 
-#: config.py:99
+#: config.py:104
 msgid ""
 "Enable this option for nonfiction books and novels that have character pages "
 "on Wikipedia/Fandom"
 msgstr ""
 "Activez cette option pour les livres non-fiction et nouvelles qui ont des "
 "pages de personnage sur Wikipedia/Fandom"
 
-#: config.py:111
+#: config.py:116
+msgid "Run spaCy with GPU"
+msgstr "Exécuter spaCy sur le GPU"
+
+#: config.py:119
+msgid ""
+"Requires CUDA. GPU will be used when creating X-Ray file if spaCy has "
+"transformer model for the book language with ner component."
+msgstr ""
+"Requiert CUDA. Le GPU sera utilisé à la création du fichier X-Ray si spaCy "
+"inclut un modèle transformer ayant le composant ner pour le langage de ce "
+"livre."
```

Comment:
I changed these two texts on the master branch in this [commit](https://github.com/xxyzz/WordDumb/commit/ca9afe49d0694e56e93f391b396f228ba4b3a639)(after the v3.28.0 tag) to highlight CUDA requirement, so the code uses these texts now: https://github.com/xxyzz/WordDumb/blob/ca9afe49d0694e56e93f391b396f228ba4b3a639/config.py#L116-L121

You can make new changes to the `patch-6` branch and new commits will show up in this pull request, no need to close the pull request. If you're not sure how to add commits to a branch I can apply the change for you.

Reply:
Please add the changes for me ...me? sure of anything with GitHub :-)

Reply:
I pushed a commit to your branch. I moved the "Requiert CUDA" text to match the English text. Thanks for your contribution!